tests/ipfwd/test_nhop_count.py

```diff
@@ -0,0 +1,178 @@
+import os
+import pytest
+import time
+import logging
+
+from collections import namedtuple
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.plugins.loganalyzer.loganalyzer import LogAnalyzer
+
+pytestmark = [
+    pytest.mark.topology('t1', 't2')
```

Comment:
why not t0 ?

Reply:
t0's will not have more paths than 16. Not sure if there's any value in adding t0 topology.